gix-url/tests/fixtures/make_baseline.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# list of urls that should be tested for all platforms
+tests=()
+# urls only intended for testing on Unix platforms
+tests_unix=()
+# urls only intended for testing on Windows
+tests_windows=()
+
+# The contents and structure of this loop are a direct copy
```

Comment:
Could you call it `adaptation` instead of `direct copy`? It's more of a wording nitpick to avoid giving people the impression GPL code was copied here - which after checking it was definitely not.